src/test/java/games/strategy/engine/framework/startup/mc/GameSelectorModelTest.java

```diff
@@ -105,8 +101,7 @@ private void verifyTestObjectObserverUpdateSent() {
   }
 
   private void prepareMockGameDataExpectations() {
-    when(mockGameData.getGameVersion()).thenReturn(mockVersion);
-    when(mockVersion.toString()).thenReturn(fakeGameVersion);
+    when(mockGameData.getGameVersion()).thenReturn(new Version(fakeGameVersion));
```

Comment:
This change was triggered by making `Version` `final`, which itself was triggered by Equals Verifier.  Making `Version` `final` is a reasonable change since it is effectively a value type.

However, once `Version` is `final`, it can no longer be mocked.  The mocking of `Version` in this test seemed unreasonable anyway, as we were essentially subverting the `Version` class invariants by using an invalid version string.  I simply replaced the test value with a valid, but unlikely, version string.

Reply:
Okay, though may as well store `Version` as a class variable.
I'm not sure about subversion of the class invariant, though it does not seem there is a stronger guarantee of valid data and so completely bogus data in a test case will better exercise the code to demonstrate that we handle this data (and also demonstrates that the code does not have proper invariant checks in place and that our test data *could* look like this, which is for sure a smell. The fix is to add an invariant that then requires this test to be udpated to have stronger data. I generally will use the absolute worst case data I can think of for a test and still have the code worked. It can show the lack of integrity checks in the code)

Reply:
> Okay, though may as well store `Version` as a class variable.

Not sure I follow here.  The `Version` instance isn't used outside of this method.  Why would we promote it to class level?  Contrast this with the `Integer` instantiated two lines below.  It's basically the same usage.  Should that be extracted to a field, as well?

> I'm not sure about subversion of the class invariant, though it does not seem there is a stronger guarantee of valid data

There is such an invariant in place: the `Version(String)` constructor will raise an exception if you pass `"fakeGameVersion"`.

Reply:
> The Version instance isn't used outside of this method. 

Good point, meanwhile we do use the inner String value elsewhere. I'm okay with leaving as is then.